torch/testing/_internal/common_device_type.py

```diff
@@ -1970,13 +1970,8 @@ def get_all_device_types() -> list[str]:
     return ["cpu"] if not torch.cuda.is_available() else ["cpu", "cuda"]
 
 
-# skip since currently flex attention requires at least `avx2` support on CPU.
 IS_FLEX_ATTENTION_CPU_PLATFORM_SUPPORTED = (
-    not torch.xpu.is_available()
-    and not torch.cuda.is_available()
-    and not IS_MACOS
-    and torch.cpu._is_avx2_supported()
-    and os.getenv("ATEN_CPU_CAPABILITY") != "default"
+    not torch.xpu.is_available() and not torch.cuda.is_available() and not IS_MACOS
```

Comment:
Hi @malfet, I have removed the restriction for AVX2. Now the Flex Attention is not X86-only on CPU. Do you think it is OK to merge this PR now?